tests/features/asset_data_flow/__init__.py

```diff
@@ -1,137 +0,0 @@
-
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (testing):** Entire test suite removed for asset data flow.

The removal of the entire test suite for asset data flow might leave the related functionality untested. If the API client removal made these tests obsolete, consider adding new tests that cover the updated functionality or integration.